src/frametree.cpp

```diff
@@ -819,13 +820,19 @@ vector<SplitMode> SplitMode::modes(SplitAlign align_explode, SplitAlign align_au
 int FrameTree::splitCommand(Input input, Output output)
 {
     // usage: split t|b|l|r|h|v FRACTION
-    string splitType, strFraction;
+    string splitType, strFraction = "0.5";
     if (!(input >> splitType )) {
         return HERBST_NEED_MORE_ARGS;
     }
     bool userDefinedFraction = input >> strFraction;
-    double fractionFloat = userDefinedFraction ? atof(strFraction.c_str()) : 0.5;
-    int fraction = FrameSplit::clampFraction(FRACTION_UNIT * fractionFloat);
+    FixPrecDec fraction = FixPrecDec::fromInteger(0);
+    try {
+        fraction = Converter<FixPrecDec>::parse(strFraction);
+    }  catch (const std::exception& e) {
+        output << "invalid argument: " << e.what() << endl;
```

Comment:
I think most (all?) of our error messages start with a capital letter. I'd suggest sticking to that style here.

Reply:
Oh and maybe even be more explicit about the problem. I recall being frustrated whenever a tool tells me that it doesn't like my input, but does not tell me what the problem is. For example, it could say "Cannot parse as decimal number: " instead.

Reply:
> I think most (all?) of our error messages start with a capital letter. I'd suggest sticking to that style here.

It's not entirely consistent. On `master`, we have:
```bash
$ grep -ni 'invalid argument' *.cpp
frametree.cpp:565:        output << "invalid argument: " << e.what() << endl;
monitor.cpp:370:                ": Invalid argument \"" << argv[1] << "\"\n";
rulemanager.cpp:253:                    HSWarning("Invalid argument \"%s\" for rule consequence \"%s\": %s\n",
tag.cpp:283:        output << "invalid argument: " << e.what() << endl;
```

Reply:
And I'd like to add that I also want to simplify the argument parsing anyway, because I'm re-writing the same code over and over again. You're partly right with your second comment: the error message says what's wrong with the argument (e.g. "only digits after '.'"), but it doesn't say *which* argument could not be parsed. But I'd address both these two issues in a separate PR.